### ISSUE TITLE
Remove pair-programming project from 101

### ISSUE THREAD

**Issue by I3uckwheat** [TheOdinProject/discussions#27]
Not many people find partners for this, and we get a lot of questions about it. Maybe changing some worded to make it obviously optional could help.

**Comment by codyloyd** [comment]
or just remove it entirely

pairing is cool but we need a system to actually make it work.

**Comment by codyloyd** [comment]
Voting Comment for https://github.com/TheOdinProject/discussions/blob/master/proposals/remove-pair-programming.md

reply with a 👍 or 👎  to approve or reject this proposal

**Comment by Saranoya** [comment]
I'm one person who did find a pair for this project initially, but then had Real Life happen, which derailed the collaboration. Even despite not having a pair anymore, though, I returned to the project later for two reasons: 

1. I thought it could teach me some useful skills I didn't have yet, and I turned out to be right. If nothing else, this project forced me to realize that setInterval in JS is not like sleep in Ruby, in that it doesn't really 'pause' the execution of the entire script. Without Pomodoro, I still wouldn't know that at this point. 

2. The finished product is actually a useful tool that I personally use while working on other projects. 

Because of this, even though the decision may already have been made, I'd like to enter a belated vote for *not* removing this project, but rather updating the lesson to make it clear that pairing is entirely optional.
